pkg/controller/node.go

```diff
@@ -82,13 +83,13 @@ func (r *ReconcileNode) Reconcile(ctx context.Context, request reconcile.Request
 	}
 
 	// If the node has no DeletionTimestamp, it means it's either create event or update event
-	if n.ObjectMeta.DeletionTimestamp.IsZero() && !n.Spec.Unschedulable {
+	if n.ObjectMeta.DeletionTimestamp.IsZero() {
 		// Add the new schedulable node in availableAttachmentsMap
-		r.addNodeToAvailableAttachmentsMap(ctx, n.Name, n.GetLabels())
-
-		// Node is schedulable, proceed to attempt creation of replica attachment
-		logger.Info("Node is now available. Will requeue failed replica creation requests.")
-		r.tryCreateFailedReplicas(ctx, nodeavailability)
+		if added := r.addNodeToAvailableAttachmentsMap(ctx, n.Name, n.GetLabels()); added {
```

Comment:
np: "added" -> "ok" because we will return true if the element is already in the map